src/metrics.ts

```diff
@@ -0,0 +1,64 @@
+import { collectDefaultMetrics, Counter, Gauge, register } from 'prom-client';
+
+if (process.env.NODE_ENV == 'production') {
+  register.clear();
+  collectDefaultMetrics({ prefix: 'peribolos_' });
+}
+
+export const totalNumOfReposServing =
+  (register.getSingleMetric(
+    'peribolos_total_num_of_repos_serving'
+  ) as Gauge<string>) ||
+  new Gauge({
+    name: 'peribolos_total_num_of_repos_serving',
+    help: 'Total numbers of repositories serving.',
```

Comment:
Not sure if we want this metric. From what I understand, this is reporting the total of currently active installs based on received installs and uninstalls...  This will be true only for current session on controller restart this will be reset to 0 anyways. I think it's better to calculate this from total_installs - total_uninstalls overtime via prom query. WDYT?